src/System.Linq.Expressions/tests/Unary/UnaryIsFalseNullableTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Xunit;
+
+namespace Tests.ExpressionCompiler.Unary
+{
+    public static class UnaryIsFalseNullableTests
+    {
+        #region Test methods
+
+        [Fact] //[WorkItem(3196, "https://github.com/dotnet/corefx/issues/3196")]
```

Comment:
Pieces of prior art with regards to referencing issues:
- Introduce a WorkItemAttribute a la Roslyn
- WinHttpHandlerTest.cs uses a comment with the URI
- Environment.GetEnvironmentVariable.cs uses ActiveIssueAttribute, not sure what will happen with the annotation after it gets fixed